samples/nrf9160/lwm2m_client/src/ui/ui_led.c

```diff
@@ -59,10 +41,9 @@ int ui_led_pwm_on_off(uint8_t led_num, bool new_state)
 
 	state[led_num] = new_state;
 
-	ret = pwm_pin_set_usec(pwm_leds[led_num].dev, pwm_leds[led_num].channel,
-			       PWM_PERIOD_USEC,
-			       pulse_width[led_num] * new_state,
-			       pwm_leds[led_num].flags);
+	ret = pwm_set_dt(&pwm_leds[led_num],
+			 PWM_USEC(PWM_PERIOD_USEC),
+			 pulse_width[led_num] * new_state);
```

Comment:
what units `pulse_width[led_num]` is in? Should we add `PWM_USEC`, too?

Reply:
Yes, wrapped in `PWM_USEC`.